pkg/compose/create.go

```diff
@@ -892,9 +892,10 @@ func isUnixAbs(path string) bool {
 
 func buildMount(project types.Project, volume types.ServiceVolumeConfig) (mount.Mount, error) {
 	source := volume.Source
+	sourceTrimmed := strings.TrimPrefix(source, project.Name+"_")
```

Comment:
you should not have to do this. compose model is expected to be consistent, so project prefix should be added to all items in `project.Volumes` and resource references.